content/en/about/faq/setup/install-method-selection.md

```diff
@@ -23,34 +23,37 @@ The following lists some of the pros and cons of each of the available methods:
     - The `istioctl` command can set values automatically based on your running environment,
       thereby producing varying installations in different Kubernetes environments.
 
-1. [istioctl manifest generate](/docs/setup/install/istioctl/#generate-a-manifest-before-installation)
+1. [Install using Helm](/docs/setup/install/helm/)
```

Comment:
(The part above this implicitly still recommends `istioctl` as the best install method.  We cool with that?)

Reply:
okay look craig I'm trying not to get myself in trouble here, that's why I left it like it was :D

if we did flip the implicit rec, that should be a separate PR anyway.

Reply:
In my head the recommended order should be (and currently is, since this doesn't change that)


- `istioctl`
- `helm`
- YOLOing with templates (lots of drawbacks, which tool you use for this largely doesn't matter, go with God and don't call us)